applications/reference-data/src/main/java/no/dcat/themes/service/LosRDFImporter.java

```diff
@@ -192,4 +168,33 @@ private LosNode getByURI(URI keyword, List<LosNode> allLosNodes) {
         }
         return null;
     }
+
+    List<LosNode> importFromLosSource() {
+        final Model model = ModelFactory.createDefaultModel();
+
+        model.read(LOS_URL);
+
+        List<Resource> concepts = model.listResourcesWithProperty(RDF.type, SKOS.Concept).toList();
+
+        // Extract the theme tree with words.
+        List<LosNode> allLosNodes = concepts.stream()
+                .filter(r -> isInScheme(r, LOS_ONTOLOGY.WORD) || isInScheme(r, LOS_ONTOLOGY.THEME))
+                .map(LosRDFImporter::extractLosItemFromModel)
+                .collect(Collectors.toList());
+
+        // Generate the paths.
+        for (LosNode node : allLosNodes) {
+            node.setLosPaths(generateLosPaths(node, allLosNodes));
+        }
+
+        logger.debug("Got {} LOSes", allLosNodes.size());
+
+        return allLosNodes;
+
+    }
+
+    private static class LOS_ONTOLOGY {
+        static private final Resource WORD = ResourceFactory.createResource("http://psi.norge.no/los/ontologi/ord");
+        static private final Resource THEME = ResourceFactory.createResource("http://psi.norge.no/los/ontologi/tema");
```

Comment:
Those hard-coded links may be a potential deal-breaker. Is there no Java-way to select words and topics?